cg_hermes/config/balsamic.py

```diff
@@ -237,6 +244,41 @@
         "is_mandatory": True,
         "used_by": ["audit"],
     },
+    frozenset(RAW_TAGS["multiqc_picard_AlignmentSummaryMetrics.json"]): {
+        "tags": ["qc-metrics", "multiqc", "picard-alignment"],
```

Comment:
@jemten mentioned the qc-metrics was previously used, would this not cause any clashes?

Reply:
But maybe it's a good idea to include `janus` (and remove vogue) as a valid `used_by` tag

Reply:
I am unsure about housekeeper tags; I have not worked with them and do not entirely know about the principles in how we set them up. If you think there is merit to that then we can definetly do it.

Intuitively, I would say that we can indeed remove vogue as the project is dead.

Reply:
Done here: https://github.com/Clinical-Genomics/hermes/pull/96